### PR TITLE
Add jquery-xmlrpc@0.4.3 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10575
Related issue(s): # #

@cdnjs/intern3 Please help me on this PR. Thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/timheap/jquery-xmlrpc
 * Official website (optional, not the repository): https://jquery-xml-rpc.readthedocs.org/
 * License and its reference: unlicense, [ref](https://github.com/timheap/jquery-xmlrpc/blob/master/LICENCE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 10
   - Count of stars: 73
   - Count of forks: 23

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
Checked!
@pvnr0082t Please help me review this PR. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 

>This one looks close the the minimum requirement, we can wait for it.

As @PeterDaveHello said in https://github.com/cdnjs/cdnjs/issues/10575#issuecomment-279607829, this lib still does not reach the popularity threshold in cdnjs.
Should we continue this PR or waiting for notable popularity? cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Let's just get it online, just be more careful next time.

File: ajax/libs/jquery-xmlrpc/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "jQuery",
+    "XML-RPC"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/timheap/jquery-xmlrpc.git"
+  },
+  "homepage": "https://jquery-xml-rpc.readthedocs.io/en/latest/",
+  "author": "timheap",
+  "license": "unlicense",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-xmlrpc/package.json]: **U**nlicense

File: ajax/libs/jquery-xmlrpc/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-xmlrpc",
+  "filename": "jquery.xmlrpc.min.js",
+  "version": "0.4.3",
+  "description": "A jQuery library for talking to XML-RPC services from JavaScript",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-xmlrpc/package.json]: @hare1039 Why not use the info from upstream? `description`, `keywords` and `author`. 
ref: https://github.com/timheap/jquery-xmlrpc/blob/v0.4.3/xmlrpc.jquery.json

File: ajax/libs/jquery-xmlrpc/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "timheap",
+  "license": "unlicense",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/timheap/jquery-xmlrpc.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.xmlrpc*.js"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-xmlrpc/package.json]: We can just use `"jquery.xmlrpc*"`

Comment (hare1039) [inline ajax/libs/jquery-xmlrpc/package.json]: @pvnr0082t I fixed these problem. Please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 You don't need to create a hyperlink for license reference in PR comment next time.

**Comment by PeterDaveHello** [comment]
Where is the homepage info come from? It doesn't look like a homepage, just a doc.

**Comment by timheap** [comment]
@PeterDaveHello It does not have a home page, only the docs page and the repo.

**Comment by PeterDaveHello** [comment]
@timheap yeah so we can just remove the info to mark it as a homepage

**Comment by hare1039** [comment]
@pvnr0082t I removed the wrong homepage. Please review this PR again. Thanks.

**Pull request merged**